lib/src/models/generic_contract/contract_updates_subscription.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'contract_updates_subscription.freezed.dart';
+
+part 'contract_updates_subscription.g.dart';
+
+@freezed
+class ContractUpdatesSubscription with _$ContractUpdatesSubscription {
+  const factory ContractUpdatesSubscription({
+    @JsonKey(includeIfNull: false) bool? state,
```

Comment:
```bool state``` is not a good combination of type and variable name because it can mean anything. IMHO.

Reply:
kk, i'll change it in next pr

Reply:
Fixed in #62